src/services/crud/users/logic.py

```diff
@@ -1,12 +1,43 @@
 from .models import User
-from src.libs.data_manager import DataManager
+# from src.libs.data_manager import DataManager
 
```

Comment:
Chú ý: với toàn bộ các api còn lại, phải check các đối tượng tồn tại trước mới làm. nếu không tồn tại thì raise exception 404, tương tự với các trường hợp lỗi khác, tươngr tượng nó có thể xảy ra thì phải xử lý
xem cách fastapi xử lý lỗi: https://fastapi.tiangolo.com/tutorial/handling-errors/